src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -508,48 +511,39 @@ const EffectiveDateViewField = ({ formData }) => {
 };
 
 const AddressViewField = ({ formData }) => {
-  const {
-    addressLine1, addressLine2, addressLine3, city, state,
-    zipCode, militaryStateCode, militaryPostOfficeTypeCode
-  } = formData;
+  const { addressLine1, addressLine2, addressLine3, city, state, country, zipCode } = formData;
```

Comment:
Yeah, that makes sense

Reply:
Oh wait, I know why I did this. They're `addressLine1` in EVSS's swagger. I suppose we could have vets-api rename them if we care.

Reply:
Yup, we should have vets-api rename them because we want to stick to as close a replica of the common address schema as we can. I believe @SMLuthi is sending up `street1` and friends in pre-fill, and submit should also transform them.

Reply:
This has actually already been changed to `addressLine1`, `addressLine2` and `addressLine3`. I *can* change it to street again but if its just for convention sake, I'm not sure if its worth the time. But I can be persuaded.